host/sgx/calls.c

```diff
@@ -390,6 +431,19 @@ static oe_result_t _handle_ocall(
     if (arg_out)
         *arg_out = 0;
 
+    if (func != OE_OCALL_LOG)
+    {
+        oe_log_args_t log_args;
```

Comment:
Why not `oe_log`? Isn't that the more high-level function to use?

Reply:
No reason other than my lack of familiarity with the codebase, will use oe_log

Reply:
Done